### ISSUE TITLE
fallback to <body> by default seems too invasive...

### ISSUE THREAD

**Issue by okliv** [unpoly/unpoly#160]
I would suggest another enhancement(as for me): explicit fallback selector setup instead of default `<body>` replacement in case of `up-target` html element absence.

i found a config option to neutralise that behaviour:

````js
up.fragment.config = {fallbacks: ['']}
````

but i think, that this has to be inverted to explicit `fallbacks` stack mention

````js
up.fragment.config = {fallbacks: ['body','browser','laptop','apartment','house','locality','country','continent','planet']}
````

Especially, in addition to precisely informative footer error rendering
![image](https://user-images.githubusercontent.com/884634/114095131-beae3800-98c5-11eb-8d57-554479a3ab34.png)

It looks like excessive double "punishment" for a regular fault during development at least because by default we loose whole page to make sure that our target or fallback fragment exists in DOM or play with css to prepare it, for example...

**Comment by triskweline** [comment]
I'm afraid I didn't understand your feature request. Can you rephrase it?

**Comment by okliv** [comment]
If i use unpoly out of the box, set up-target=".replace-me" and do not have <div class="replace-me">...</div> in my html (no matter why) it replaces whole <body>. it confuses. i propose to "relax" unpoly behaviour here.

**Comment by triskweline** [comment]
The toast overlay for fatal errors is removed in Unpoly 2.

There are going to be multiple ways to disable fallbacks in Unpoly 2, but `body` is going to remain a default fallback target in the stock distribution.

The reason fallbacks were introduced is that the server might render unexpected content (like a login form from an expired session, or an error message). The user should see that content rather than having the link fail silently.

**Comment by okliv** [comment]
disagree. toast overlay looks on its place. body replacement still looks too much aggressive. but you are the boss.
